### ISSUE TITLE
[pyclustering.cluster.xmeans.xmeans.get_centers()] issue

### ISSUE THREAD

**Issue by himanshu94** [annoviko/pyclustering#388]
The number of allocated centers given by xmeans.get_centers() is not consistent with the number of clusters given by the xmeans.get_clusters(). In my case xmeans.get_cluster() is giving 15 list which means 15 clusters are present while the xmeans.get_centers() is giving 26 centers which should be same and my kmax  is set to 20 and the number of centers I am getting by xmeans.get_centers is 26.

**Comment by annoviko** [comment]
@himanshu94,

Thank you for the created issue. Could you please share data that you have used and code?

**Comment by annoviko** [comment]
@himanshu94 , I have found out the root cause of the bug, soon I will provide the correction.

**Comment by annoviko** [comment]
@himanshu94 , 

Bug has been corrected, correction is available on branch 0.7.dev that can be obtained by the following command:
```bash
git clone https://github.com/annoviko/pyclustering.git . -b 0.7.dev
```

The correction will be available in the next release.

**Comment by himanshu94** [comment]
Thanks @annoviko. Sorry for the late reply
